survey/survey-web-adaptor/src/main/java/me/nalab/survey/web/adaptor/findfeedback/response/feedback/BookmarkResponse.java

```diff
@@ -0,0 +1,22 @@
+package me.nalab.survey.web.adaptor.findfeedback.response.feedback;
+
+import java.time.LocalDateTime;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@Builder
+@ToString
```

Comment:
```suggestion
@Getter
@Builder
@ToString
@EqaulsAndHashCode
```
중요한거는 아닌데, 
`@EqualsAndHashCode` 요거 dto랑 도메인에 다 넣어주기로 한거같아
controller.response라 나중에 기억나면 리팩토링해도 버그 없을듯?!